src/planscape/collaboration/urls.py

```diff
@@ -16,4 +17,9 @@
         GetInvitationsForObject.as_view(),
         name="get_invitations",
     ),
+    path(
+        "update_invitation/<int:object_pk>",
```

Comment:
I don't know if we want to follow a url pattern like the one above and just depend on REST verbs to differentiate the function to call, but since this is only changing the role, I'm also not sure if the URL should indicate this limitation. It's easy to change, of course.